pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-section-index"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
My projects are maintained in a way that does not accept a version change within a pull request.